src/types/SortField.ts

```diff
@@ -0,0 +1,8 @@
+export enum SortField {
+  Name = 'name',
+  Sex = 'sex',
+  Born = 'born',
+  Died = 'died',
+  Mother = 'mother',
+  Father = 'father',
+}
```

Comment:
It can be just array

Reply:
Not fixed, it can be just array of strings, and it should be not it `types` folder, but in constants file